config/default.yaml

```diff
@@ -16,13 +16,14 @@ services:
       authorization: '/o/authorize'
       token: '/o/token'
       endSession: '/o/endSession'
+      invite: '/api/v2/invitations'
     systemUser:
       username: 'idam-user-dashboard-system-user@user.idam'
       password: 'Pa55word11'
     clientID: 'idam_user_dashboard'
     clientSecret: 'idam_user_dashboard_secret'
     callbackURL: 'http://localhost:3100/oauth2/callback'
-    scope: 'openid profile roles manage-user search-user create-user'
+    scope: 'profile roles manage-user search-user create-user create-invitation'
```

Comment:
I don't think users logging into User Dashboard should be given that scope (for now). 
Maybe we need separate configuration for the scopes required for the client_credentials authentication.

Reply:
Addressed in https://github.com/hmcts/idam-user-dashboard/pull/440/commits/ae420a46043915790f0454563342272b2f8f7591